src/App.vue

```diff
@@ -21,26 +21,39 @@
 import Header from '@/components/Header'
 import ModuleNav from '@/components/ModuleNav'
 import Footer from '@/components/Footer'
+import { mapActions } from 'vuex'
 
 export default {
   // set initial state here
   // save in state in store
   data () {
     return {
       loggedIn: true,
-      authenticated: true
+      authenticated: true,
+      studentIdHardCoded: 94
```

Comment:
This is a good idea to add the student id here where we can easily change it if needed for demo purposes. (Of course once/if we get auth working we won't need it anymore.)

Reply:
yep, it's easier to change up the user here.